application-1/analytics/deployment.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     app.kubernetes.io/component: backend
     app.kubernetes.io/instance: user-analytics
     app.kubernetes.io/version: "1.0"
+  annotations:
+    app.openshift.io/vcs-uri: "git@github.com:redhat-developer/topology-example.git"
+    app.openshift.io/vcs-ref: "v1.0.0"
```

Comment:
Could you add a different ref ? Something more realistic?

Reply:
I have put commit hash - 
d1e1433